src/test/java/core/basesyntax/MainAppTest.java

```diff
@@ -1,5 +1,22 @@
 package core.basesyntax;
 
 public class MainAppTest {
+    public static void main(String[] args) {
 
+        Machine excavator = new Excavator();
+        excavator.setName("excavator");
+
+        Machine bulldozer = new Bulldozer();
+        bulldozer.setName("bulldozer");
+
+        Machine truck = new Truck();
+        truck.setName("truck");
+
+        Machine[] machines = new Machine[]{excavator, bulldozer, truck};
+
+        for (Machine machine : machines) {
+            machine.doWork();
+            machine.stopWork();
+        }
+    }
```

Comment:
The `MainAppTest` class seems to be a misnomer as it does not contain any test methods or assertions. Consider renaming it to reflect its purpose more accurately or implementing actual test methods using a testing framework like JUnit.